packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -426,6 +436,17 @@ abstract class RepositoryBase extends Resource implements IRepository {
     });
   }
 
+  public notifyOnPullRequestMerged(
```

Comment:
Change `notifiyOnPullRequestMerged()` to simply call this new method, instead of duplicating its body.